torch/csrc/utils/tensor_types.cpp

```diff
@@ -36,9 +41,11 @@ at::Type& type_from_string(const std::string& str) {
     }
   });
 
+#ifndef NO_PYTHON
   if (str == "torch.Tensor") {
     return torch::tensor::get_default_tensor_type();
   }
+#endif
```

Comment:
What's the problem here?

Reply:
The function returns an object of type defined here: https://github.com/pytorch/pytorch/blob/master/torch/csrc/tensor/python_tensor.cpp#L22

Should we create a stub for `PyTypeObject` so that we can call this function?

Reply:
Is this file even needed in NO_PYTHON builds? It looks like it's a bunch of helpers that are fairly specific to Python